{"name": "test_getbyid_failure", "status": "passed", "start": 1729322914125, "stop": 1729322914134, "uuid": "e12df18d-7e18-4f7d-8c2a-42861eec967c", "historyId": "440d2917e2e3c9db6ce38bcef72ba8ca", "testCaseId": "440d2917e2e3c9db6ce38bcef72ba8ca", "fullName": "Exercise.exercise_test.TestExerciseServiceWithMock#test_getbyid_failure", "labels": [{"name": "parentSuite", "value": "Exercise"}, {"name": "suite", "value": "exercise_test"}, {"name": "subSuite", "value": "TestExerciseServiceWithMock"}, {"name": "host", "value": "DESKTOP-IDGJ8R7"}, {"name": "thread", "value": "21312-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Exercise.exercise_test"}]}